src/main/java/com/netflix/imflibrary/st2067_2/Application2E2021.java

```diff
@@ -0,0 +1,338 @@
+        new Fraction(30),
+        new Fraction(30000, 1001),
+        new Fraction(50),
+        new Fraction(60),
+        new Fraction(60000, 1001),
+        new Fraction(120)
+    };
+
+    /* Table 3 at SMPTE ST 2067-21:2023 */
+    static final CharacteristicsSet[] IMAGE_CHARACTERISTICS = {
+        new CharacteristicsSet(
+            1920,
+            1080,
+            Arrays.asList(Colorimetry.Color1, Colorimetry.Color2, Colorimetry.Color3),
+            Arrays.asList(8, 10),
+            Arrays.asList(FrameLayoutType.SeparateFields),
+            Arrays.asList(FPS_HD),
+            Arrays.asList(Sampling.Sampling422),
+            Arrays.asList(Quantization.QE1),
+            Arrays.asList(ColorModel.YUV)
+        ),
+        new CharacteristicsSet(
+            1920,
+            1080,
+            Arrays.asList(Colorimetry.Color1, Colorimetry.Color2, Colorimetry.Color3),
+            Arrays.asList(8, 10),
+            Arrays.asList(FrameLayoutType.FullFrame),
+            Arrays.asList(FPS_HD),
+            Arrays.asList(Sampling.Sampling422),
+            Arrays.asList(Quantization.QE1),
+            Arrays.asList(ColorModel.YUV)
+        ),
+        new CharacteristicsSet(
+            1920,
+            1080,
+            Arrays.asList(Colorimetry.Color1, Colorimetry.Color2, Colorimetry.Color3),
+            Arrays.asList(8, 10, 12),
+            Arrays.asList(FrameLayoutType.FullFrame),
+            Arrays.asList(FPS_HD),
+            Arrays.asList(Sampling.Sampling444),
+            Arrays.asList(Quantization.QE1),
+            Arrays.asList(ColorModel.YUV, ColorModel.RGB)
+        ),
+        new CharacteristicsSet(
+            3840,
+            2160,
+            Arrays.asList(Colorimetry.Color4),
+            Arrays.asList(8, 10),
+            Arrays.asList(FrameLayoutType.FullFrame),
+            Arrays.asList(FPS_UHD),
+            Arrays.asList(Sampling.Sampling422),
+            Arrays.asList(Quantization.QE1),
+            Arrays.asList(ColorModel.YUV)
+        ),
+        new CharacteristicsSet(
+            3840,
+            2160,
+            Arrays.asList(Colorimetry.Color3),
+            Arrays.asList(8, 10, 12, 16),
+            Arrays.asList(FrameLayoutType.FullFrame),
+            Arrays.asList(FPS_UHD),
+            Arrays.asList(Sampling.Sampling422),
+            Arrays.asList(Quantization.QE1),
+            Arrays.asList(ColorModel.YUV)
+        ),
+        new CharacteristicsSet(
+            3840,
+            2160,
+            Arrays.asList(Colorimetry.Color5, Colorimetry.Color8),
+            Arrays.asList(10, 12),
+            Arrays.asList(FrameLayoutType.FullFrame),
+            Arrays.asList(FPS_UHD),
+            Arrays.asList(Sampling.Sampling422),
+            Arrays.asList(Quantization.QE1),
+            Arrays.asList(ColorModel.YUV)
+        ),
+        new CharacteristicsSet(
+            3840,
+            2160,
+            Arrays.asList(Colorimetry.Color5, Colorimetry.Color7),
```

Comment:
Color5 is covered in previous set.
```suggestion
            Arrays.asList(Colorimetry.Color7),
```